lib/surface_bulma/dropdown.ex

```diff
@@ -0,0 +1,62 @@
+defmodule SurfaceBulma.Dropdown do
+  use Surface.LiveComponent
+
+  @moduledoc """
+  An interactive dropdown menu for discoverable content.
+
+  [Bulma Docs - Dropdown](https://bulma.io/documentation/components/dropdown/)
+  """
+
+  @doc "Is always open"
+  prop is_active, :boolean
+
+  @doc "Activates on hover"
+  prop is_hoverable, :boolean
+
+  @doc "Left/Right aligned (left by default)"
+  prop is_right, :boolean, default: false
+
+  @doc "Menu appears above/below (below by default)"
+  prop is_up, :boolean, default: false
+
+  slot current_item, required: true
```

Comment:
What do you think about renaming this to "trigger"?

Reply:
From what is highlighted it is unclear which you mean: `is_up`/`drop_up` or the `current_item` slot. I don't see how either would be appropriately labeled as "trigger". Can you elaborate?

Reply:
The `current_item` slot, this is called the trigger in the Bulma documentation. I figured that since the dropdown item does not change when an item is selected, a `Dropdown.Item` cannot become the `Dropdown.CurrentItem` and it is therefore confusing to make it seem like it could be (this was how I read it initially).

Reply:
Sorry, I see now that there is also a `Dropdown.Trigger` component. Maybe it would make sense to modify that to have a slot where the trigger text can go?

Reply:
I think I was misinterpreting the context that this would be used. I was thinking it would benefit from the flexibility of being able to change the content of the "current" item via the user's own implementation tracking its state (i.e. using it like a dropdown select). But in reading your response, and reviewing how it is demonstrated in the Bulma docs, it would seem that it's more for use cases such as dropdown navigations (where the "current item" doesn't change).

Reply:
Okay, I can see that reasoning. I was thinking of the "current_item" being something that may be interchanged in certain circumstances, such as this being used as a sort of dropdown-select. But I can appreciate that this may not be the intended use case for this component. You're probably right, especially to maintain some parity with the Bulma labels.